scripts/update.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 declare APP_NAME='testcenter'
+declare IS_TC_UP=false
```

Comment:
Mir ist nicht ganz klar warum die Variable notwendig ist um den state der Container zu tracken. Werden in der create_backup() nicht immer am Anfang immer die DB und BE des TC hochgefahren und am Ende wieder alle Container runtergefahren?

Reply:
Nein, das Update kann sowohl beim laufenden als heruntergefahrenen System durchgeführt weren.
Die Variable stellt denn Ausgangszustand wieder her.